dna/problems.py

```diff
@@ -339,6 +347,7 @@ def score(self, predictions_by_group, targets):
         top_1_regrets = []
         top_k_regrets = []
         top_k_counts = []
+        # TODO: ndcg@k
```

Comment:
Can we add ndcg to the subset problem? I don't have time right now, but I may later this evening.

Reply:
Sure, I don't see why not.  We don't have to use it if we don't want to.